core/roboconf-core/src/main/java/net/roboconf/core/Constants.java

```diff
@@ -85,7 +85,7 @@
 	/**
 	 * The <strong>rules</strong> file extension.
 	 */
-	String FILE_EXT_RULES = ".rules";
+	String FILE_RULES = "rules.cfg";
```

Comment:
Why do we need the *.cfg extension?
Measures files have a *.measures extension.